docs/update_and_migration/from_2.5/to_3.3.md

```diff
@@ -91,6 +91,14 @@ Resolve the conflicts in the following way:
     It is impossible to update an Enterprise edition (`ezsystems/ezplatform-ee`)
     to an [[= product_name_content =]] edition.
 
+    Make sure that composer.json has the following `repositories` entry:
```

Comment:
```suggestion
    It is impossible to update an Enterprise edition (`ezsystems/ezplatform-ee`)
    to an [[= product_name_content =]] edition.

    Make sure that `composer.json` has the following `repositories` entry:
```

Are these two statements logically related? Or rather they are two separate cautions?

If they are not connected, I would add "Also, " before "make sure that ..."